tests/benchdnn/graph/ref_primitive.cpp

```diff
@@ -244,8 +245,8 @@ void ref_primitive_t::check_correctness(
         check_buffer_overwrite(args.dnn_mem(i), args.arg(i), res);
 
         const auto arg = args.arg(i);
-        const auto &mem_dt = args.find(arg);
-        const auto &mem_fp = args_.find(arg);
+        const dnn_mem_t &mem_dt = args.find(arg);
+        const dnn_mem_t &mem_fp = args_.find(arg);
```

Comment:
Why is this change needed? What's the issue with `auto`?

Reply:
@dzarukin Hi Dmitry, thanks for having a look into this. I have replaced auto with explicit data types for improved readability, consistency, and type safety. When debugging it was particularly useful to have the data type specified and I did go an extra mile checking that the function returns the same type. Should not be a problem but I am happy to review it if it is a deal breaker.

Reply:
It's fine with me, just wanted to be sure there's no compiler/code issue involved. Generally, all `dnn_mem_t` are either named as mem_XXX or m_XXX or plain mem/m if that helps for future debugging.

Happy to approve. BTW, QQ: do you have Graph API working on aarch64?

Reply:
I’ve been working on this the past few days, and all tests for the Graph API should run fine. There are a few edge cases that fail but I’ll detail in an issue soon and shall be fixed in the foreseeable future. 

Thanks for the approval!